lib/wongi-engine/dsl.rb

```diff
@@ -84,6 +84,9 @@ module Wongi::Engine::DSL
     clause :greater
     accept Wongi::Engine::GreaterThanTest
 
+    clause :greaterthanorequal
```

Comment:
a `gte` alias would be good for readability

Reply:
agreed

Reply:
done